portfolio/src/main/webapp/index.html

```diff
@@ -5,15 +5,245 @@
+          </a>
+          <a href="https://mlt.org/">
+          <img src="https://m1.fluidreview.com/media/assets/reviewrooms/New-ML4T/logo/Clear_MLT_Logo-_PNG.png" class="org__logo" alt="Company 1" />
+          </a>
+          <a href="https://www.allstarcode.org/">
+          <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQhDoawl4lkop5F8WY9j-hBvV8KgTSBLmD_FA&usqp=CAU" class="org__logo" alt="Company 3" />
+          </a>
+          <a href="https://www.cs.stonybrook.edu/About-Us/Student-Resources">
+          <img src="https://media0.giphy.com/media/xnudw3yfNlvml1ztcB/giphy.gif" class="org__logo" alt="Company 1" />
+          </a>
+        </div>
+      </div>
+    </section>
+
+    <!-- ***** About ***** -->
+
+    <section class="about" id="about">
+      <div class="row">
+        <h2>About Me</h2>
+        <div class="about__content">
+          <div class="about__text">
+            <p>
+              I am a first generation college student from an Ecuadorian family.
+              I was born and raised in Queens, New York. As a full time student, I 
+              get to pursue two of my biggest passions. My passion for technology and
+              my passion for helping students from underrepresented backgrounds in achieving
+              their dreams of pursuing a career in STEM. In my free time, I enjoy listening
+              to music, binge watching shows, and performing card magic for friends and family.
+            </p>
+            
+            <a href="https://padlet.com/justinpoveda/iyinihj3xsoqi8oi" class="btn">My Resume</a>
+          </div>
+
+          <div class="about__photo-container">
+            <img class="about__photo" src="/Images/HeadShot.jpeg" alt="" />
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+
+  <!-- ***** Contact ***** -->
+
+  <section class="contact" id="contact">
+    <div class="row">
+      <h2>Get in Touch</h2>
+      <div class="contact__info">
+        <p>
+          Thank you for checking out my portfolio! I hope you enjoyed seeing 
+          me work and got to learn a bit about who I am. Whether you are a friend/
+          family member, a recruiter or anyone, I am more than happy to connect
+          and chat about anything. Literally anything, whether it's about music, tech,
+          advise, opportunities, card magic, TV shows, and the list goes on. Or you just 
+          want to say "Hi 👋" in any case feel free to let me know. I will do my best to
+          respond back 😊. The quickest way to reach out to me is via an email.
+            
+          <p> Phone Number: 347-645-6202   -  Email: justin.poveda@stonybrook.edu  -  Queens, NY</p>
+        </p>
+        <a href="mailto:justin.poveda@stonybrook.edu" class="btn">Send Me An Email</a>
```

Comment:
I would recommend leaving this as regular text rather than a mailto link. For people who don't have a default mail client set up, it can be unexpected and annoying for a website to cause a program to start on their computer.